tests/service_tests/modules/injectors/test_metadata_injection.py

```diff
@@ -6,6 +6,8 @@
 import pandas as pd
 import pytest
 
+pytest.importorskip("mrc")
+
 from nv_ingest.framework.orchestration.morpheus.modules.injectors.metadata_injector import on_data
```

Comment:
We should figure this out too. If we're keeping the old morpheus code, we should leave its unit tests in place. If we're going to strip all of it out, then we should remove tests for Morpheus specific construts.